notary-btc-integration-test/src/integration-test/kotlin/integration/btc/BtcMultiWithdrawalIntegrationTest.kt

```diff
@@ -189,6 +188,18 @@ class BtcMultiWithdrawalIntegrationTest {
                 transactionOutput
             ) == btcAddressDest
         })
+        //Check that change addresses are watched
+        withdrawalEnvironments.forEach { withdrawalEnvironment ->
+            val changeAddresses = withdrawalEnvironment.btcChangeAddressProvider.getAllChangeAddresses().get()
```

Comment:
use `map` instead of `get()`?

Reply:
I was just too lazy to use it in tests. Anyway, `Exception` will be thrown.